packages/store/src/symbols.ts

```diff
@@ -147,3 +153,13 @@ export interface NgxsAfterBootstrap {
 }
 
 export type NgxsLifeCycle = Partial<NgxsOnInit> & Partial<NgxsAfterBootstrap>;
+
+export type FunctionCall = <T>(func: (...args: any[]) => T) => T;
```

Comment:
Rename to 

```ts
type Callback<T = any, U = any> = (...args: U[]) => T;
```

Reply:
Not necessarily a callback. It is to represent a function parameter to a higher order function.

Reply:
Got rid of this because it is not used. Made signature simpler so that this is not necessary as well.